x-pack/plugins/siem/server/lib/timeline/routes/__mocks__/import_timelines.ts

```diff
@@ -163,13 +163,6 @@ export const mockParsedTimelineObject = omit(
   mockUniqueParsedObjects[0]
 );
 
-export const mockConfig = {
-  get: () => {
-    return 100000000;
```

Comment:
This value is an order of magnitude larger than the value in the [mock config in detection engine directory](https://github.com/elastic/kibana/pull/63430/files?w=1#diff-f4f55d98b7e9b99fd7bb244254ef28c5R21). Not sure if that serves a purpose for the tests on timelines but just wanted to point it out.

https://github.com/elastic/kibana/pull/63430/files?w=1#diff-12565563a255e05c185b90d50337ec51L48